### PR TITLE
COG-498 AuthorizeService refactoring: main part 

### PR DESCRIPTION
**Pull request by AlexanderSerbin**
### JIRA Issue Link
https://osi-cwds.atlassian.net/browse/COG-498

### Technical Description
Refactoring

### Tests
- [ ] I have included unit tests 
- [ ] I have included integration tests 
- [ ] I have included performance tests 
- [ ] I have included other tests 
- [x] I have NOT included tests 
Since it's refactoring no additional tests added

### Types of changes
- [ ] New feature (non-breaking change which adds functionality)
- [x] Refactoring (No behavioral changes)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

### CODE REVIEW

File: src/main/java/gov/ca/cwds/idm/service/authorization/AuthorizationServiceImpl.java [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+    }
+    return user;
+  }
+
+  private String getAdminStrongestRole() {
+    UniversalUserToken admin = CurrentAuthenticatedUserUtil.getCurrentUser();
+    return UserRolesService.getStrongestAdminRole(admin);
+  }
+
+  private ActionsAuthorizer createAdminForUserAuthorizer(User user) {
```
Comment (sorych) [inline src/main/java/gov/ca/cwds/idm/service/authorization/AuthorizationServiceImpl.java]: I suggest renaming to createAdminAsctionsAuthorizer() here

File: src/main/java/gov/ca/cwds/idm/service/authorization/AuthorizationUtils.java [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+import gov.ca.cwds.UniversalUserToken;
+import gov.ca.cwds.idm.dto.User;
+
+final class AuthorizationUtils {
+
+  private AuthorizationUtils() {
+  }
+
+  static boolean principalInTheSameCountyWith(User user) {
```
Comment (sorych) [inline src/main/java/gov/ca/cwds/idm/service/authorization/AuthorizationUtils.java]: how about isPrincipalInTheSameCountyWith() ?

File: src/main/java/gov/ca/cwds/idm/service/authorization/ActionsAuthorizer.java [hunk 3]
```diff
@@ -0,0 +1,13 @@
+package gov.ca.cwds.idm.service.authorization;
+
+interface ActionsAuthorizer {
```
Comment (sorych) [inline src/main/java/gov/ca/cwds/idm/service/authorization/ActionsAuthorizer.java]: maybe AdminActionsAuthorizer name will better describe this interface?

File: src/main/java/gov/ca/cwds/idm/service/authorization/AuthorizationServiceImpl.java (hunk 1 above)
Comment (AlexanderSerbin) [reply src/main/java/gov/ca/cwds/idm/service/authorization/AuthorizationServiceImpl.java]: Good catch! Renamed to suggested name.

File: src/main/java/gov/ca/cwds/idm/service/authorization/AuthorizationUtils.java (hunk 2 above)
Comment (AlexanderSerbin) [reply src/main/java/gov/ca/cwds/idm/service/authorization/AuthorizationUtils.java]: Done

File: src/main/java/gov/ca/cwds/idm/service/authorization/ActionsAuthorizer.java (hunk 3 above)
Comment (AlexanderSerbin) [reply src/main/java/gov/ca/cwds/idm/service/authorization/ActionsAuthorizer.java]: Good catch! Done

**Pull request merged**